### ISSUE TITLE
Star ligature not working in HTML/CSS?

### ISSUE THREAD

**Issue by cassidoo** [xotypeco/big_shoulders#41]
I've been trying to enable the correct ligatures to get them working in HTML and CSS based on the [documentation on design.chicago.gov](https://design.chicago.gov/typography/), with no success:

[Example of my attempts](https://codepen.io/cassidoo/pen/abQQQKW??editors=1100)

Besides setting `font-variant-ligatures: discretionary-ligatures;`, what else do I need to do to make this work? I noticed that it's not appearing properly on the [typography website](https://design.chicago.gov/typography/) either. Did something change?

**Comment by xotypeco** [comment]
it's because you're calling the fonts from the Google Fonts API. the API doesn't serve a complete font file, including dlig instructions, for reasons I can't remember. (likewise, true small caps & alternate forms aren't served by the API.)

dlig will work if you've downloaded and hosted the full font files on your own domain.

as a workaround to the API not serving dlig: I also added the rectangle and star as unicode symbols (which I think the API _will_ call).

the rectangle is filledRect (U+25ac) and the star is sixPointedBlackStar (U+2736)

the guide website isn't working, and doesn't note any of these details, because the person within the city who made it also called the fonts from the API. they didn't have a chance to document the added details because the Lightfoot administration eliminated their job during the early days of the pandemic. 

so basically, Lightfoot put out a press release saying "look at this amazing thing I did," then made it nearly impossible for anyone to support it—which was a super typical Lori Lightfoot thing to do.
